core/src/utils/notifications-service.ts

```diff
@@ -0,0 +1,122 @@
+import Toastify, { Offset } from 'toastify-js'
+
+export enum TypeIcons {
+  'success' = 'check-circle-filled',
+  'error' = 'sparkle-filled',
+  'info' = 'sparkle-filled'
+}
+
+export interface ToastOptions {
```

Comment:
We want to expose as little options as possible to a) keep it very simple, b) make it easier to switch to another library and c) make it opinionated and disallow too many variations. So please remove the following options and find sensible defaults:

- text / node should become `content: string | Node` IMO (yes, we then need some kind of mapping logic for the options).
- duration
- selector
- destination
- newWindow
- gravity / position should be combined into one position attribute, that takes stuff like top-left, top-right... 
- className
- stopOnFocus (should be true)
- callback
- offset
- style
- oldestFirst

Reply:
yes, there is not much left, but that is intentional :D